docs/reference/configuration.md

```diff
@@ -69,6 +69,38 @@ This reference uses `.` to denote the nesting of values.
 * `blockstore.s3.retention.report_s3_prefix_url` - Base S3 URL to use
   for writing batch tagging completion reports.  Must be writable by
   `blockstore.s3.retention.role_arn`.
+* `cataloger.type` (one of `rocks` or `mvcc`, default `rocks`) - whether to use `mvcc` or
+  `rocks` cataloger.  Changing from `mvcc` to `rocks` requires migration.  Changing back is
+  not possible.
+* `committed.local_cache` - an object describing the local (on-disk) cache of metadata from
+  permanent storage:
+  + `committed.local_cache.size_bytes` (`int` : 1 GiB) - bytes for local cache to use on disk.  The cache may use more storage for short periods of time.
```

Comment:
Please specify the integer value here, unless we support `1 GiB` as an actual value

Reply:
Done!  (It's as easy as `M-x calc RET 2 30 ^` to get `1073741824`).

However hardly convenient.  Eventually we should make configuration more convenient.